source/common/stats/thread_local_store.cc

```diff
@@ -178,18 +198,13 @@ StatType& ThreadLocalStoreImpl::ScopeImpl::safeMakeStat(
   if (!central_ref) {
     std::vector<Tag> tags;
     std::string tag_extracted_name = parent_.getTagsForName(name, tags);
```

Comment:
Can you add a comment explaining why it's not necessary to use the truncated name for tag extraction?